Makefile

```diff
@@ -418,24 +418,17 @@ build-volumes:
 	$(SILENT)docker volume inspect $(GOPATH_VOLUME_NAME) >/dev/null 2>&1 || docker volume create $(GOPATH_VOLUME_NAME)
 	$(SILENT)docker volume inspect $(GOCACHE_VOLUME_NAME) >/dev/null 2>&1 || docker volume create $(GOCACHE_VOLUME_NAME)
 
-.PHONY: main-builder-image
-main-builder-image: build-volumes
```

Comment:
Does `build-volumes` still need to run before dockerized builds?

Reply:
I think this depends on `BIND_GOCACHE`/`BIND_GOPATH` being `0` which is the case on `Darwin`.

Reply:
Looking back on this, I believe what I was trying to convey was what you said in your [other comment](https://github.com/stackrox/stackrox/pull/10539/files#r1539821617). Not sure why I didn't see your comment.  Regardless, I tested this PR by running `make main-build-dockerized` with `BIND_GOCACHE` and `BIND_GOPATH` set to `0`, and it worked fine.  The first run downloaded all the dependencies, and then I ran it again and observed that all the dependencies were cached in the volumes.